pkgs/development/compilers/go/2-dev.nix

```diff
@@ -40,6 +40,8 @@ let
     "armv6l" = "arm";
     "armv7l" = "arm";
     "powerpc64le" = "ppc64le";
+    "mips64" = "mips64";
+    "mips64el" = "mips64le";
```

Comment:
Please drop the changes to `2-dev.nix` from this PR. This list of aches is out of order/sync with the other `go_1_` files. 

This derivation needs several other updates to match the others, I'll do all of it in a separate PR.

Reply:
done.